### DISCUSSION TITLE
Where is the model after training finished

### DISCUSSION THREAD

**Discussion by zwsjink** [alibaba/FederatedScope#457] [General]
I follow the example on https://federatedscope.io/docs/examples/ , and play with the standalone mode. After the program finish, where can i find the model file?  I only find some log files under directory such as 
  FederatedScope/exp/FedAvg_convnet2_on_femnist_lr0.01_lstep1/sub_exp_20221206120123,  

drwxr-xr-x   7 zwsjink  staff    224 12  6 12:01 .
drwxr-xr-x  11 zwsjink  staff    352 12  6 12:01 ..
-rw-r--r--   1 zwsjink  staff   4148 12  6 12:01 config.yaml
-rw-r--r--   1 zwsjink  staff   9426 12  6 12:01 eval_results.log
-rw-r--r--   1 zwsjink  staff   2745 12  6 12:01 eval_results.raw.gz
-rw-r--r--   1 zwsjink  staff  30723 12  6 12:01 exp_print.log
-rw-r--r--   1 zwsjink  staff   4139 12  6 12:01 system_metrics.log

**Comment by xieyxclack** [comment]
Hi, you might need to specify `cfg.federate.save_to`, and the model saving happens at https://github.com/alibaba/FederatedScope/blob/master/federatedscope/core/workers/server.py#L517

**Reply by zwsjink** [reply]
thanks man, I just see your reply.
